source/class/qx/ui/table/cellrenderer/AbstractImage.js

```diff
@@ -255,10 +255,11 @@ qx.Class.define("qx.ui.table.cellrenderer.AbstractImage",
     // overridden
     _getCellAttributes : function(cellInfo)
     {
-      var tooltip = this.__imageData.tooltip;
+      const baseAttrs = this.base(arguments, cellInfo);
 
+      var tooltip = this.__imageData.tooltip;
       if (tooltip) {
-        return "title='" + tooltip + "'";
+        return `${baseAttrs} title='${tooltip}'`;
```

Comment:
Same thing here. This will change in v7, which no longer supports the generator, but it would be good if we can backport the ARIA feature

Reply:
Do we really need the backport to version 6? Should'nt we bring out v7 instead?
Or change the the not supported syntax during backport?

Backtick syntax reads much nicer...

Reply:
ARIA support should also be available for legacy apps, I think. I like backticks better, too, but here it would be so easy to stay backwards-compatible. After v7 is out, we no longer need to worry about such things, but at the moment, we still support v6.